nominal/nominal.py

```diff
@@ -339,3 +348,41 @@ def _get_start_end_timestamp_csv_file(
         IntegralNanosecondsUTC(start.to_datetime64().astype(int)),
         IntegralNanosecondsUTC(end.to_datetime64().astype(int)),
     )
+
+def create_checklist(
+    assignee_email: str,
+    title: str,
```

Comment:
everything in the client library is uniformly using "name" even if the api uses "title"